src/main/java/executor/service/service/parallelflowexecutor/impls/subscribers/ExecutionSubscriber.java

```diff
@@ -0,0 +1,42 @@
+package executor.service.service.parallelflowexecutor.impls.subscribers;
+
+import executor.service.model.service.ExecutableScenario;
+import executor.service.service.executionservice.ExecutionService;
+import executor.service.service.parallelflowexecutor.ParallelFlowExecutorService;
+import executor.service.service.parallelflowexecutor.Operatable;
+
+import java.util.List;
+
+public class ExecutionSubscriber implements Operatable {
+    private static final long DELAY = 2000L;
```

Comment:
In your code, the delay between iterations is set to 2000 milliseconds (2 second) and cannot be changed. Consider adding a configuration that allows users to set their own delay if they so desire.

Reply:
I don't want to spend time to write code of configuration static constants.
In the other option, if it set in constructor, it would adding boilerplate code, which doesn't have a lot of sense.
When app will migrate to Spring it possible to do, but discover a bicycle ...

Reply:
This application must also be done well.